CHANGELOG.md

```diff
@@ -7,27 +7,27 @@ Mapbox welcomes participation and contributions from everyone.
 For details on how v2 differs from v1 and guidance on migrating from v1 of the Mapbox Navigation SDK for Android to the v2 public preview, see [2.0 Navigation SDK Migration Guide](https://github.com/mapbox/mapbox-navigation-android/wiki/2.0-Navigation-SDK-Migration-Guide).
 
 ### Changelog
-#### Features
-- :warning: A new billing session would be started when `ArrivalObserver#onNextRouteLegStart` is triggered. [#4821](https://github.com/mapbox/mapbox-navigation-android/pull/4821)
-
 #### Bug fixes and improvements
+- Fixed an issue where the route was never refreshed because of an internal timer not starting when a new route was set. [#4831](https://github.com/mapbox/mapbox-navigation-android/pull/4831)
+- When downloading critical tiles, the Map tiles are preferred over Routing tiles to ensure that the map is visible as soon as possible. [#4826](https://github.com/mapbox/mapbox-navigation-android/pull/4826)
 - Fixed an issue where `offRoute` and `BannerInstructions` states were not immediately reset after setting a new route. [#4810](https://github.com/mapbox/mapbox-navigation-android/pull/4810)
-- Renamed `FeedbackEvent.Description` annotation to `FeedbackEvent.SubType` and Added `FeedbackHelper` with utilities for working with feedback types and subtypes [4813](https://github.com/mapbox/mapbox-navigation-android/pull/4813)
+- :warning: Renamed `FeedbackEvent.Description` annotation to `FeedbackEvent.SubType` and Added `FeedbackHelper` with utilities for working with feedback types and subtypes [4813](https://github.com/mapbox/mapbox-navigation-android/pull/4813)
```

Comment:
`Added` shouldn't be capitalised. Or maybe we should split this into two entries.